### PR TITLE
Updating instructions in examples/transports tut

### PR DESCRIPTION
**Pull request by walkerlj0**
Updated part 1  (1.js)

### CODE REVIEW

**Comment by BigLep** [comment]
2022-05-03 conversation: there are two things in this PR: 
1. moving back to CJS
2. some new/helpful content

We don't want the move back to CJS given we are making js-libp2p ESM-only.  We do want the new/helpful content.

@walkerlj0 : if you are wanting to contribute more here, please reduce to #2 only.  Otherwise, it will get picked up once the EMS-only js-libp2p is released.

**Comment by wemeetagain** [comment]
@walkerlj0 libp2p 0.37.0 has finally been released with ESM-only exports.

Can you try to make the examples run against the latest version?

File: examples/transports/README.md [hunk 1]
```diff
@@ -21,44 +21,52 @@ Then, in your favorite text editor create a file with the `.js` extension. I've
...
-// If you are familiar with TCP, you might have noticed
-// that we specified the node to listen in 0.0.0.0 and port
-// 0, which means "listen in any network interface and pick
-// a port for me
-console.log('listening on:')
-node.multiaddrs.forEach((ma) => console.log(`${ma.toString()}/p2p/${node.peerId.toB58String()}`))
+createNode().catch(err => {
+  console.error(err)
+  process.exit(1)
+})
```
Comment (wemeetagain) [inline examples/transports/README.md:69]: Can you revert the changes other than this? We want to keep examples using the latest versions of libp2p dependencies, using ESM imports, etc.

**Review by wemeetagain — CHANGES_REQUESTED** [review]

**Comment by BigLep** [comment]
@walkerlj0: are you able to incorporate the feedback so the PR can be merged?

**Comment by walkerlj0** [comment]
I will try and run it in the next coupld days, so sorry, been swamped!

**Comment by mpetrunic** [comment]
@walkerlj0 we converted it to draft state until you are ready.

**Comment by walkerlj0** [comment]
@mpetrunic tried running transports/1.js and got this error, advice on how to troubleshoot?

`file:///Users/lindsayjowalker/Git/js-libp2p/examples/transports/1.js:3
const Libp2p = require('libp2p')
               ^

ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and '/Users/lindsayjowalker/Git/js-libp2p/examples/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
    at file:///Users/lindsayjowalker/Git/js-libp2p/examples/transports/1.js:3:16
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12)`

**Comment by mpetrunic** [comment]
> @mpetrunic tried running transports/1.js and got this error, advice on how to troubleshoot?
> 
> **Note I did fetch upstream from libp2p:master
> 
> ```
> file:///Users/lindsayjowalker/Git/js-libp2p/examples/transports/1.js:3
> const Libp2p = require('libp2p')
>                ^
> 
> ReferenceError: require is not defined in ES module scope, you can use import instead
> This file is being treated as an ES module because it has a '.js' file extension and '/Users/lindsayjowalker/Git/js-libp2p/examples/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
>     at file:///Users/lindsayjowalker/Git/js-libp2p/examples/transports/1.js:3:16
>     at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
>     at async Promise.all (index 0)
>     at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
>     at async loadESM (node:internal/process/esm_loader:88:5)
>     at async handleMainPromise (node:internal/modules/run_main:65:12)
> ```

If you fetched latest master, do `npm install` in root followed by `npm run build`. Also do `npm install` in the examples directory. You shouldn't have `require` statement anywhere, if you have you probably have some local changes or you didn't pull latest libp2p:master.

Gonna close this PR as examples work just fine for me locally!

**Pull request closed without merging**